include/chopper/cereal/serialize_std_filesystem_path.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <filesystem>
+
+#include <seqan3/core/concept/cereal.hpp>
```

Comment:
We just need this because we need to define `CEREAL_SAVE_FUNCTION_NAME`, etc., right?

Reply:
On a nights (with a little more) sleep, I noticed that this is not a seqan macro but cereals. So I think I'll switch to cereal include.